### ISSUE TITLE
Evaluation on MPII dataset

### ISSUE THREAD

**Issue by ravi-dhime** [daniegr/EfficientPose#44]
hi @daniegr, 

I am trying to recreate the reported pckh@50 score. With your pretrained EfficientposeRT + Single Person MPII Validation dataset, I am getting only 62% pckh. Whereas the reported score is 82%. I might be missing one or two important steps. Can you pls help me identify them. Below are more details about my best performing experiment.

Library: PyTorch
Model: Pretrained EfficientposeRT
Data: 10% of Single Person MPII Training set
Single Person Extraction: Using scale & center from annotations. scale increased by 25%
Image Format: B, C, H, W with RGB channel order.
Image Pixel Range: 0 to 1, normalized using Imagenet stats
Image Size: 224x224
Invisible joint annotations: are ignored in pckh calculation
Test Time Augmentation: original image + horizontally flipped image are used
Heatmap: Average of the heatmaps predicted on the original & the flipped images
Threshold for the predicted heatmap values: 0.0
Keypoints: points with peak heat value are used as keypoints

**Comment by daniegr** [comment]
@ravi-dhime: Nice to get to know about your efforts to replicate our results on MPII. There is definitely something that is not well aligned with your implementation and how EfficientPose should ideally perform. First of all, regarding your preprocessing of images, are the pixel values zero centered using `tensorflow.keras.applications.imagenet_utils. preprocess_input` with 'torch' mode to perform preprocessing of raw pixel values (i.e., integers from 0 to 255), and not between 0 and 1 as you indicate above? Btw. be aware that since EfficientPose is trained on a large portion of the MPII training set, there is likely to be overfitting issues if you evaluate the model on a portion of the training set.
